src/scenes/Engagement/LanguageEngagement/Create/CreateLanguageEngagement.tsx

```diff
@@ -0,0 +1,58 @@
+import { useSnackbar } from 'notistack';
+import React from 'react';
+import { Except } from 'type-fest';
+import { ButtonLink } from '../../../../components/Routing';
+import { ProjectOverviewDocument as ProjectOverview } from '../../../Projects/Overview/ProjectOverview.generated';
+import { useCreateLanguageEngagementMutation } from './CreateLanguageEngagement.generated';
+import {
+  CreateLanguageEngagementForm,
+  CreateLanguageEngagementFormProps as Props,
+} from './CreateLanguageEngagementForm';
+
+type CreateLanguageEngagementProps = Except<Props, 'onSubmit'> & {
+  projectId: string;
+};
+
+export const CreateLanguageEngagement = ({
+  projectId,
+  ...props
+}: CreateLanguageEngagementProps) => {
+  const [createEngagement] = useCreateLanguageEngagementMutation();
+  const { enqueueSnackbar } = useSnackbar();
+  const submit: Props['onSubmit'] = async (input) => {
+    const createLanguageEngagementInput = {
+      engagement: { projectId, languageId: input.language.id },
+    };
+    const res = await createEngagement({
+      variables: { input: createLanguageEngagementInput },
+      refetchQueries: [
+        {
+          query: ProjectOverview,
+          variables: { input: projectId },
+        },
+      ],
+    });
+    const engagement = res.data!.createLanguageEngagement.engagement;
+
+    enqueueSnackbar(
+      `Created engagement for:
+${
+  engagement.language?.value?.name?.value ??
+  engagement.language?.value?.displayName?.value
+}
+        `,
+      {
+        variant: 'success',
+        action: () => (
+          <ButtonLink
+            color="inherit"
+            to={`projects/${projectId}/engagements/${engagement.id}`}
+          >
+            View
+          </ButtonLink>
+        ),
+      }
+    );
+  };
+  return <CreateLanguageEngagementForm {...props} onSubmit={submit} />;
```

Comment:
We determined earlier that we don't have a need to split out "Form" components anymore since we aren't creating stories for them.